features/202-session-management/use-cases.md

```diff
@@ -40,3 +40,10 @@ async function updateDeviceID(sessionID: string, deviceID: string): Promise<void
     await skygear.auth.updateSession(sessionID, { deviceID });
 }
 ```
+
+# Session name
+To display a session name in session management UI, it is recommended to
```

Comment:
UI/UX-wise, I think in all cases we should display device model & OS version, IP address etc.

But we display device name / session name if present as a more prominent information, and downplay other information in those cases.

Reply:
It is a recommendation on how to display a meaningful session name, developer should consider the actual UI design themselves.

Reply:
Ya of coz, but since we write a recommendation in our spec, I think we should give a sensible best practices / default?

Even with the extra information I think device model & OS version, IP address (country) etc should still be displayed.

Reply:
I agree a session entry should contains these information. However, this recommendation concerns about session name, which displaying IP address would be a bad UX.

Reply:
Well we could replace IP address with country when we have it.

Actually the point is the current wordings imply we should not display device Model OS etc if user supplied session name / device name is present.)